libs/ui/src/bmd_paper_ballot.dual_language.test.tsx

```diff
@@ -333,12 +332,12 @@ describe('English ballot style', () => {
   const contests = getContests({ ballotStyle: englishBallotStyle, election });
 
   test('all votes filled in', async () => {
-    const votes: VotesDict = _(contests)
-      .keyBy((c) => c.id)
-      .mapValues((c) =>
-        c.type === 'yesno' ? generateYesNoVote(c) : generateCandidateVotes(c)
-      )
-      .value();
+    const votes: VotesDict = Object.fromEntries(
```

Comment:
I'm sort of intentionally making working with `Record` and `Dictionary` types not that nice because of the [issues with them we discussed in Slack](https://votingworks.slack.com/archives/C014MGC0WRE/p1701725954463349).